tests/Filesystem/FilesystemTest.php

```diff
@@ -339,7 +339,7 @@ public function testSharedGet()
             $result *= $status;
         }
 
-        $this->assertTrue($result === 1);
+        $this->assertFalse($result === 1);
```

Comment:
Yeah we need more explanation as to why the test was breaking in the first place.